src/gulpfile.js

```diff
@@ -29,6 +29,18 @@ gulp.task('build:zip', (done) => {
 });
 
 /**
+ * Does a rebuild of your theme's locale files and runs translation
+ * tests on each file using @shopify/theme-lint
+ *
+ * @function test
+ * @memberof slate-cli.tasks
+ * @static
+ */
+gulp.task('test', (done) => {
+  runSequence('build:locales', 'lint:locales', done);
```

Comment:
1. Good call - not sure why I went the route of linting the locales in the `dist` folder.
2. I removed `build:locales` altogether, since `build:locales` was just a subset of `build:assets` - so we can do the same thing there.

Also adding a watch gulp task for testing for locales.